MathConverter/MathConverter.csproj

```diff
@@ -1,97 +1,14 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project>
+  <Sdk Name="Microsoft.NET.Sdk.WindowsDesktop" />
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{C53EC42B-978E-4667-A9A9-6401793267B7}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <TargetFrameworks>netcoreapp3.1;net45;</TargetFrameworks>
+    <UseWPF>true</UseWPF>
     <RootNamespace>HexInnovation</RootNamespace>
     <AssemblyName>MathConverter</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <TargetFrameworkProfile>Client</TargetFrameworkProfile>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>

```

Comment:
Before I saw this PR i had done some similar work. We can eliminate the need for the .tt in the main project by allowing it to generate the assembly info and then using an element like this

```
<Copyright>
      Copyright © Hex Innovation $([System.DateTime]::Now.ToString(`yyyy`))
</Copyright>
```

The demo project also depends on that tt file and something similar could be done there. 

To avoid the duplicate attribute error I just deleted the attributes from AssemblyInfo.cs except the xml namespace ones and the internalsVisibleTo.

Reply:
Didn't know that. Thank you for this hint.